### ISSUE TITLE
Corrupted Extrinsic dataset

### ISSUE THREAD

**Issue by MingyuKim-2933** [tigerchen52/LOVE#6]
Hi,

I was wondering When generating typos for Extrinsic tasks (tables 3, 4) in the paper,
Did you corrupt both the train and test datasets?

Thank you!

**Comment by tigerchen52** [comment]
Hi,

Only the test datasets are corrupted to check the robustness.

Best,

Lihu

**Comment by MingyuKim-2933** [comment]
Thank you for your quick and kind answer! :)

**Comment by MingyuKim-2933** [comment]
I have one more question..
If you use dev dataset for hyperparameter tuning, did you corrupted dev datasets, too?
I apologize for asking frequently.

Thank you!

**Comment by tigerchen52** [comment]
In fact, only the test set is corrupted. Since it is possible to have unseen words in real-world scenarios, and we aim to measure the authentic robustness of language models. 
Your questions are welcome :-D

**Comment by MingyuKim-2933** [comment]
Thank you for your kind answer!! :)
